app/webpacker/controllers/sticky_controller.js

```diff
@@ -15,7 +15,7 @@ export default class extends Controller {
     this.containerTarget.style.bottom = "-1px";
     const observer = new IntersectionObserver(
       ([e]) => {
-        e.target.classList.toggle("sticked", e.intersectionRatio <= 1);
+        e.target.classList.toggle("sticked", e.intersectionRatio < 1);
```

Comment:
I don't understand, but assume you have tested this out to work out the correct rule ;)

Reply:
As an aside, I reckon this kind of thing can be handled by a third-party library/module, which would help remove complexity from our codebase and may be more reliable or efficient. But perhaps that's already been considered and this was best solution (it's quite simple at least!)

Reply:
Yep. It's always a dilemma to know if we should use a third party library :
 - Yep, this third party is great, but my need is very simple, I don't want to import and use this huge lib.
 - I should use a lib, instead of reinventing the wheel.

I think the main key is to be ok to rethink about it constantly. 

Here, needs are very simple, so let's keep it for now, and see if we need to change it!